deb-package-builder/new_extension.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
```

Comment:
I think you can use this shell script for creating a initial skeleton of new extensions.

Can you add comments showing an example for adding one?
Do you need manual tweaks? If so, can you also add what kind of modifications will be required?